lib/wraft_doc/search/typesense.ex

```diff
@@ -0,0 +1,34 @@
+defmodule WraftDoc.Search.Typesense do
+  @moduledoc """
+  Handles operations for Typesense
+  """
+
+  alias WraftDoc.Search.Encoder
+
+  def create_collection(schema) do
+    ExTypesense.create_collection(Encoder.to_document(schema))
+  end
+
+  def create_document(document, collection_name) do
+    document = Encoder.to_document(document)
+    typesense_document = Map.merge(document, %{collection_name: "#{collection_name}"})
+    ExTypesense.create_document(typesense_document)
+  end
+
+  def get_document(id, collection_name) do
+    ExTypesense.get_document("#{collection_name}", to_string(id))
```

Comment:
```suggestion
    ExTypesense.get_document(collection_name, to_string(id))
```

Since `collection_name` is already a string, why do we have to string interpolate it?